src/components/App/App.js

```diff
@@ -29,9 +38,9 @@ class App extends Component {
           <h1>Putrid Portabellos</h1>
         </header>
         {this.state.selectedMovie ? (
-        <MovieInfo selectMovie ={this.state.selectedMovie}/>
+        <MovieInfo clickMovieSelect ={this.clickMovieSelect} selectedMovieId = {this.state.selectedMovie}/>
         ) : (
-        <MovieContainer movies = {this.state.allMovies.movies} clickMovieSelect={this.clickMovieSelect}/>)}
+        <MovieContainer movies = {this.state.allMovies} clickMovieSelect={this.clickMovieSelect}/>)}
         <FooterForm />
```

Comment:
Consider if you might be able to pull this ternary logic out of the return and into a variable before the return.   Then you could cleanly access that variable within the return.   We want to strive to keep our return statements as clean and logic-free as possible.